aiohttp/payload.py

```diff
@@ -101,7 +101,7 @@ def register(self,
 
 class Payload(ABC):
 
-    _size = None  # type: Optional[float]
+    _size = None  # type: Optional[int]
```

Comment:
Yeah, I am pretty sure that making this `float` in the beginning was an err on my side.

Reply:
Agree, `3.5` bytes will never happen.